### PR TITLE
[Frontend] Added Validation in the Add FAQ Form

### PR DESCRIPTION
**Pull request by mrunalvilas**
## [Frontend] Add Validation in the Add FAQ Form

 Closes: #538

## Proposed changes:
Added Validations to the Add FAQ form from Admin Dashboard.

### Brief description of what is fixed or changed
1. Validation that checks if the question and answer fileds are empty or not.
2. Ensure that the length of the question and answer fields is 8.

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes do not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of a change in the user interface
1.

2.
screencapture-localhost-3000-dashboard-2021-03-20-21_08_24

### CODE REVIEW

File: frontend/src/pages/Admin/Components/Faq/AddFaq/add-faq.module.scss [hunk 1]
```diff
@@ -12,6 +12,12 @@
   flex: 1;
 }
 
+.validation {
+  color: red;
+  margin-top: 0;
+}
+
+
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/Faq/AddFaq/add-faq.module.scss:20]: please leave only single line gap here

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @mrunalvilas As we are using `joi-validator` for validation purpose. So please implement the validation here as well using the same inorder to be consistent. Also the validation doesn't seem to be working, please check it once on your end

**Comment by mrunalvilas** [comment]
https://user-images.githubusercontent.com/60320511/111899152-b72b0a00-8a50-11eb-8c69-b6f4dd015bae.mp4

here is a working video of the applied changes
1. I have implemented a similar validation function as is used in the sign-up field from the footer.
I would replace it with joi-validator (just want some reference in which field is it used )
2. I have replaced 2 line spaces with one line space in the add-faq.module.css file.

**Comment by Kajol-Kumari** [comment]
Hey @mrunalvilas We need the validation's error message to appear just below the respective input fields(just like we are doing it for other forms). Also as we don't want the default type error messages to appear anymore, so please remove them

**Comment by Kajol-Kumari** [comment]
> Hey @mrunalvilas We need the validation's error message to appear just below the respective input fields(just like we are doing it for other forms). Also as we don't want the default type error messages to appear anymore, so please remove them

Hey @mrunalvilas can you please provide an update on this?

**Comment by mrunalvilas** [comment]
Yes,  will update shortly

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @mrunalvilas can you please fix the padding of the error msg. It should appear near to the input box (Please see the other forms for reference)

Rest looks good to go :)

**Review by Kajol-Kumari — APPROVED** [review]
Thanks for the quick fix @mrunalvilas 
This looks good to go now :)

**Pull request merged**